sdk/storage/Azure.Storage.Common/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 12.5.0-preview.1 (Unreleased)
-
+## 12.4.1 (2020-05)
+- Fixed bug where BlobBaseClient.DownloadAsync() could corrupt downloaded data if intermittent network failure occurs.
```

Comment:
Should probably change this to ```This release contains bug fixes to improve quality.``` since these classes don't exist in Common.

Reply:
done.